### PR TITLE
Sunset: Enable maintenance mode for signup/login

### PR DESCRIPTION
**Pull request by aidvu**
- We want to disable account management while making changes.
- Show users appropriate message when maintenance mode is on.
- Show the same message if users stumble on /signup. This is for cases where users manually enter the link. Should be very rare, so no need for special care there.

Fixes: https://github.com/Automattic/delphin/issues/1169

### CODE REVIEW

File: app/reducers/user/selectors.js [hunk 1]
```diff
@@ -2,7 +2,10 @@ export const isLoggedOut = state => ! state.user.isLoggedIn && ! state.user.conn
 
 // isLoggedIn !== ! isLoggedOut, because there is a point when the app boots when
 // the app is determining if the user is logged in or out.
-export const isLoggedIn = state => state.user.isLoggedIn;
+export const isLoggedIn = state => {
```
Comment (klimeryk) [inline app/reducers/user/selectors.js]: Nice!

**Comment by aidvu** [comment]
Actually a great idea! Will try and hide `Log In`.

**Comment by aidvu** [comment]
Not going to fix the linter @mikeshelton1503. :D

Tho I'll remove the `yarn.lock` from your commit. Or at least revert. :P

**Comment by mikeshelton1503** [comment]
Oh my bad, I don't even know what that is.

**Comment by klimeryk** [comment]
:shipit: Let's deploy this to staging until we are ready to deploy to production.

**Pull request merged**